app/views/measures/_measure.html.erb

```diff
@@ -25,13 +25,14 @@
           declarable_id: declarable.short_code %>
       </span>
     <% end %>
-
     <% if measure.additional_code.present? %>
       <% if measure.measure_type.prohibitive? %>
         <%= render 'measures/additional_codes/prohibitive', measure: measure %>
       <% else %>
         <%= render 'measures/additional_codes/regular', measure: measure %>
       <% end %>
+    <% elsif !measure.additional_code.present? && measure.vat? %>
```

Comment:
Possibly this is my misunderstanding but aren't their 3 types of VAT additional code?

I don't think they're all meant to be labelled `Standard Rate`